content/guides/developer-guides/delfin/_index.md

```diff
@@ -31,33 +31,183 @@ Delfin is an Infrastructure Management framework developed in Python programming
 SODA Delfin project already contains some [drivers](https://github.com/sodafoundation/delfin/tree/master/delfin/drivers), which can be used as reference by the new third party driver developers.
 
 Existing Delfin Drivers for reference:
-  - fake_driver - This is a dummy/sample driver used for testing purpose
-  - Huawei - This driver implements Huawei's OeanStor backend
-  - VMAX - This driver implements Dell EMC's VMAX storage backend
+- fake_driver - This is a dummy/sample driver used for testing purpose
+- Huawei - This driver implements Huawei's OeanStor backend
+- VMAX - This driver implements Dell EMC's VMAX storage backend
 
 ### Code changes needed
```

Comment:
Please give the driver folder and file structure example and explain it?
Like:
/SampleDriver/
                      setup.py
                      /...
                      /...

Reply:
Path is provided in the 'Code changes needed' steps